.github/workflows/automated-release.yaml

```
@@ -0,0 +1,138 @@
+            echo Creating branch "$base_branch"
+            git checkout -b "$base_branch"
+            git push origin
+          fi
+
+      - name: Prepare release
+        id: prepare-release
+        run: |
+          echo "Updating release.yaml with desired versions"
+
+          VERSION=${{ github.event.inputs.kuadrantOperatorVersion }} \
+          AUTHORINO_OPERATOR_VERSION=${{ github.event.inputs.authorinoOperatorVersion }} \
+          DNS_OPERATOR_VERSION=${{ github.event.inputs.dnsOperatorVersion }} \
+          LIMITADOR_OPERATOR_VERSION=${{ github.event.inputs.limitadorOperatorVersion }} \
+          CONSOLE_PLUGIN_VERSION=${{ github.event.inputs.consolePluginVersion }} \
+          WASM_SHIM_VERSION=${{ github.event.inputs.wasmShimVersion }} \
+          OLM_CHANNEL=${{ github.event.inputs.olmChannel }} \
+          yq eval '(.kuadrant-operator.version = strenv(VERSION)) |
+            (.dependencies.authorino-operator = strenv(AUTHORINO_OPERATOR_VERSION)) |
+            (.dependencies.dns-operator = strenv(DNS_OPERATOR_VERSION)) |
+            (.dependencies.limitador-operator = strenv(LIMITADOR_OPERATOR_VERSION)) |
+            (.dependencies.console-plugin = strenv(CONSOLE_PLUGIN_VERSION)) |
+            (.dependencies.wasm-shim = strenv(WASM_SHIM_VERSION)) |
+            (.olm.default-channel = strenv(OLM_CHANNEL)) |
+            (.olm.channels[0] = strenv(OLM_CHANNEL))' --inplace ./release.yaml
+
+          echo "release.yaml file updated as follows:"
+
+          cat ./release.yaml
+
+          echo "Running prepare-release script"
+
+          make prepare-release
+          
+      - name: Create Pull Request
+        id: cpr
+        if: ${{ !env.ACT }}
+        uses: peter-evans/create-pull-request@v7
+        with:
+          token: ${{ secrets.KUADRANT_DEV_PAT }}
+          commit-message: Prepare release ${{ github.event.inputs.kuadrantOperatorVersion }}
+          committer: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
+          author: ${{ github.actor }} <${{ github.actor_id }}+${{ github.actor }}@users.noreply.github.com>
+          signoff: true
+          base: ${{ env.BASE_BRANCH }}
+          branch: release-v${{ github.event.inputs.kuadrantOperatorVersion }}
+          delete-branch: true
+          title: '[Release] Kuadrant Operator v${{ github.event.inputs.kuadrantOperatorVersion }}'
+          body: |
+            The following PR for the release candidate of Kuadrant Operator version ${{ github.event.inputs.kuadrantOperatorVersion }} includes:
+            - Authorino Operator version ${{ github.event.inputs.authorinoOperatorVersion }}
+            - DNS Operator version ${{ github.event.inputs.dnsOperatorVersion }}
+            - Limitador Operator version ${{ github.event.inputs.limitadorOperatorVersion }}
+            - Console Plugin version ${{ github.event.inputs.consolePluginVersion }}
+            - WASM Shim version ${{ github.event.inputs.wasmShimVersion }}
```

Small thing that bugs me is the list being out of order. Does not need to be changed but would be nice to see.
```suggestion
            - Authorino Operator version ${{ github.event.inputs.authorinoOperatorVersion }}
            - Console Plugin version ${{ github.event.inputs.consolePluginVersion }}
            - DNS Operator version ${{ github.event.inputs.dnsOperatorVersion }}
            - Limitador Operator version ${{ github.event.inputs.limitadorOperatorVersion }}
            - WASM Shim version ${{ github.event.inputs.wasmShimVersion }}
```

---

It is in order, just not alphabetical order ;)